universal-application-tool-0.0.1/app/views/admin/programs/ProgramIndexView.java

```diff
@@ -75,33 +75,24 @@ private Tag renderNewProgramButton() {
         .asButton();
   }
 
-  public ProgramDefinition getDisplayProgram(ImmutableList<ProgramDefinition> programs) {
-    Preconditions.checkState(!programs.isEmpty());
-    Optional<ProgramDefinition> draftProgram =
-        programs.stream().filter(p -> p.lifecycleStage().equals(LifecycleStage.DRAFT)).findAny();
+  public ProgramDefinition getDisplayProgram(
+      Optional<ProgramDefinition> draftProgram, Optional<ProgramDefinition> activeProgram) {
     if (draftProgram.isPresent()) {
       return draftProgram.get();
     }
-    Optional<ProgramDefinition> activeProgram =
-        programs.stream().filter(p -> p.lifecycleStage().equals(LifecycleStage.ACTIVE)).findAny();
-    if (activeProgram.isPresent()) {
-      return activeProgram.get();
-    }
-    return programs.stream().findAny().get();
+    return activeProgram.get();
   }
 
   public Tag renderProgramListItem(
-      String name, ImmutableList<ProgramDefinition> programs, Http.Request request) {
-    // TODO: Move Strings out of here for i18n.
```

Comment:
Suggestions (non-blocking): would you mind keeping this comment?

Reply:
Oh, these are admin strings and we made the decision not to internationalize them, IIRC.